packages/ios-app/js/pages/topic.js

```diff
@@ -4,7 +4,7 @@ import React from "react";
 import { NativeModules } from "react-native";
 import TopicPage from "@thetimes/topic";
 
-const { onArticlePress } = NativeModules.NativeModuleTopicActions;
+const { onArticlePress } = NativeModules.NativeModuleArticleActions;
```

Comment:
shouldn't this renaming be reflected in native land as well?

Reply:
@fourlastor yeah ! I have this PR in review ... https://github.com/newsuk/nu-digital-projectd-times-smartphone-ios/pull/1177

Reply:
I see, makes sense 👍